src/GHC.hs

```diff
@@ -29,7 +29,7 @@ defaultKnownPackages =
     , filepath, genapply, genprimopcode, ghc, ghcBoot, ghcBootTh, ghcCabal, ghci
     , ghcPkg, ghcPrim, ghcTags, haddock, haskeline, hsc2hs, hoopl, hp2ps, hpc
     , hpcBin, integerGmp, integerSimple, iservBin, libffi, mkUserGuidePart
-    , parallel, pretty, primitive, process, rts, runGhc, stm, templateHaskell
+    , pretty, process, rts, runGhc, templateHaskell
```

Comment:
I have rebased the PR and did the easy modifications you proposed. I still have to figure out how to exclude _parallel_, _primitive_ and _stm_. I will have a look at it in the next days